SimPEG/electromagnetics/base_1d.py

```diff
@@ -83,19 +125,16 @@ def __init__(
         muMap=None,
         h=None,
         hMap=None,
-        eta=0.0,
-        tau=1.0,
-        c=0.5,
-        dchi=0.0,
-        tau1=1.0e-10,
-        tau2=10.0,
+        cole_cole_parameters: ColeColeParameters | None = None,
         hankel_filter="key_101_2009",
```

Comment:
We could eventually also use an `Enum` here to restrict to the shortlist of valid inputs.